### ISSUE TITLE
Auto add user of computer

### ISSUE THREAD

**Issue by AircraftRu** [fusioninventory/fusioninventory-for-glpi#2032]
Please make this functionality:
After the inventory is automatically import from the LDAP and added to the user of computer by data of the alternate username.
Thank you!

![glpi - user - mozilla firefox 2017-01-18 18 59 40](https://cloud.githubusercontent.com/assets/17082164/22071678/f03a5c2a-ddb0-11e6-8aa1-015d5e3c20b1.png)

**Comment by ddurieux** [comment]
inventory is automatically import from the LDAP ???

**Comment by AircraftRu** [comment]
Yes, if we correctly undestand each other. Inventory is automatically import USER from the LDAP, sorry

**Comment by ddurieux** [comment]
No it import them, they must be imported to be defined when inventory come in GLPI

**Comment by AircraftRu** [comment]
Is it possible to implement initiation to import a new user by a login?
Login contained in the fiel Alternate username.

**Comment by ddurieux** [comment]
It can be heavy and use many resources to do it in inventory, be best method is to use the GLPI script to import often your LDAP users into GLPI

**Comment by AircraftRu** [comment]
That is, it is better to implement in GLPI and not in your plugin?

**Comment by AircraftRu** [comment]
I realy need this function! Need information on phone and full name of the owner computer.

**Comment by wawax** [comment]
Hello,

Why not to script ldap user import using scripts/ldap_mass_sync.php script ?

**Comment by AircraftRu** [comment]
@wawax Hello!
The script is unstable
